hazelcast/test/src/ClientTest.h

```diff
@@ -88,6 +91,50 @@ class ClientTest : public ::testing::Test
         return entries;
     }
 
+    template<typename Value = int,
+             typename Key = int,
+             typename ValueGenerator = generator<Value>,
+             typename KeyGenerator = key_int_generator>
+    std::unordered_map<Key, Value> populate_map_via_rc(
+      std::function<std::string(const Value&)> fn,
+      const std::string& map_name,
+      const std::string& cluster_id,
+      int n_entries = 10,
+      ValueGenerator&& value_gen = ValueGenerator{},
+      KeyGenerator&& key_gen = KeyGenerator{})
+    {
+        std::unordered_map<Key, Value> entries;
+        entries.reserve(n_entries);
+
+        generate_n(
```

Comment:
std namespace can be used before generate_n and inserter methods.

Reply:
I intentionally don't use those qualifiers, if they are unnecessary